pyproject.toml

```diff
@@ -8,57 +8,54 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">3.9.7, <3.12"
-openai = ">=1.0.0"
```

Comment:
why is openai removed as a package dependency? this is required for the sdk to make openai + azure calls @bufferoverflow